config/locales/en.yml

```diff
@@ -357,6 +357,7 @@ Project attributes and sections are defined in the <a href=%{admin_settings_url}
       project_roles: "Project roles"
       wp_shares: "Work package shares"
       groups: "Groups"
+    no_modify_on_shared: "You currently cannot modify or remove shared memberships through the member page. Use the sharing modal instead."
```

Comment:
This translation was completely missing and it is not being used. Added it for future use.